packages/driver/src/cy/commands/xhr.js

```diff
@@ -200,7 +200,7 @@ const startXhrServer = (cy, state, config) => {
     },
 
     onFixtureError (xhr, err) {
-      err = $errUtils.cypressErr(err)
+      err = $errUtils.cypressErr({ message: err })
```

Comment:
When there was a problem with fixture, server returns `404` with `__error` field, which contains stack-trace (string). The value was passed here, but `cypressErr` helper expects that argument will be an object with `message` field. That's why I've wrapped `err` (stack trace string) with object `{ message }`.